src/components/InlineSvg/SVG/AiAgentGradient.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { AiAgent } from './AiAgent';
+
+export function AiAgentGradient(): JSX.Element {
```

Comment:
export const AiAgentGradient: FC
```suggestion
export const AiAgentGradient: FC = () => 
```